persistent/Database/Persist/Types/Base.hs

```diff
@@ -295,12 +295,39 @@ data ForeignDef = ForeignDef
     , foreignRefTableDBName        :: !DBName
     , foreignConstraintNameHaskell :: !HaskellName
     , foreignConstraintNameDBName  :: !DBName
+    , foreignFieldCascade          :: !FieldCascade
```

Comment:
Adding a field to a data type is a breaking change, so this PR will be for the 2.11.0 release.